File: APIMASH_RottenTomatoes_StarterKit/APIMASHLib/APIMASHLib/APIMASHEvent.cs
using System;

/*
* LICENSE: https://raw.github.com/apimash/StarterKits/master/LicenseTerms-SampleApps%20.txt
*/

/*
 *
 *  A P I   M A S H
 *
 *  Update the classes here to invoke the RESTful API call(s)
 */

namespace APIMASHLib
{
    public class APIMASHEvent : EventArgs
    {
        public APIMASHStatus Status { get; set; }

        public string APIName { get; set; }

        public string Message { get; set; }

        public object Object { get; set; }
    }
}

File: Windows Phone Starter Kits/APIMASH_CNorris_VB_StarterKit_Phone/APIMASHVBLib/APIMASHEvent.vb
'
' LICENSE: https://raw.github.com/apimash/StarterKits/master/LicenseTerms-SampleApps%20.txt
'

'
'
'  A P I   M A S H
'
'  Update the classes here to invoke the RESTful API call(s)
'
Namespace APIMASHLib
    Public Class APIMASHEvent
        Inherits EventArgs
        Public Property Status() As APIMASHStatus
            Get
                Return m_Status
            End Get
            Set(value As APIMASHStatus)
                m_Status = Value
            End Set
        End Property
        Private m_Status As APIMASHStatus

        Public Property APIName() As String
            Get
                Return m_APIName
            End Get
            Set(value As String)
                m_APIName = Value
            End Set
        End Property
        Private m_APIName As String

        Public Property Message() As String
            Get
                Return m_Message
            End Get
            Set(value As String)
                m_Message = Value
            End Set
        End Property
        Private m_Message As String

        Public Property [Object]() As Object
            Get
                Return m_Object
            End Get
            Set(value As Object)
                m_Object = Value
            End Set
        End Property
        Private m_Object As Object
    End Class
End Namespace

